### PR TITLE
fix: allow disabling mfs preload from config

### PR DESCRIPTION
**Pull request by achingbrain**
Use the `preload.enabled` config key to turn on/off periodic mfs preloading.

### CODE REVIEW

File: test/core/mfs-preload.spec.js [hunk 1]
```diff
@@ -56,4 +72,22 @@ describe('MFS preload', () => {
       })
     })
   })
+
+  it('should disable preloading MFS', function (done) {
+    this.timeout(80 * 1000)
```
Comment (alanshaw) [inline test/core/mfs-preload.spec.js]: Surely this test doesn't need a timeout adjustment?

Comment (achingbrain) [reply test/core/mfs-preload.spec.js]: Have removed it.

**Comment by alanshaw** [comment]
Would you mind rebasing for green CI please?

**Comment by achingbrain** [comment]
Done

**Pull request merged**